include/net/gptp.h

```diff
@@ -56,6 +56,49 @@ struct gptp_uscaled_ns {
 	u64_t low;
 } __packed;
 
+/**
+ * @brief Extended Timestamp
+ */
+struct gptp_extended_ts {
```

Comment:
We have the net_ptp_time defined in include/net/ptp_time.h, what about creating `struct net_ptp_extended_time` in that file instead?

Reply:
Changed